apps/allocations/app/components/Card/Budget.js

```diff
@@ -0,0 +1,213 @@
+            <IconProhibited />
+            <ActionLabel>Deactivate</ActionLabel>
+          </ContextMenuItem>
+        </ContextMenu>
+      </MenuContainer>
+      <CardTitle theme={theme}>{name}</CardTitle>
+      <StatsContainer>
+        <StyledStats>
+          <StatsValueBig theme={theme}>
+            {displayCurrency(BigNumber(amount))}
+            <Text>{' ' + currency + ' per period'}</Text>
+          </StatsValueBig>
+          <StatsValueBig css={{ paddingTop: '24px' }} theme={theme}>
+            <ProgressBar
+              color={String(theme.accentEnd)}
+              value={BigNumber(allocated).div(amount).toNumber()}
+            />
+          </StatsValueBig>
+          <StatsValueSmall css={{
+            color: theme.content,
+            paddingTop: '8px',
+          }}>
+            {displayCurrency(BigNumber(amount).minus(allocated))}
+            <Text>{' ' + currency + ' below limit'}</Text>
+          </StatsValueSmall>
+          <StatsValueSmall css={{
+            color: theme.contentSecondary,
+            paddingTop: '4px',
+          }}>
+            {BigNumber(amount)
+              .minus(allocated)
+              .div(amount)
+              .multipliedBy(100)
+              .dp(0)
+              .toString()}
+            <Text>{'% remaining'}</Text>
+          </StatsValueSmall>
+        </StyledStats>
+      </StatsContainer>
+    </StyledCard>
+  )
+}
+
+Budget.propTypes = {
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  amount: PropTypes.string.isRequired,
+  currency: PropTypes.string.isRequired,
+  allocated: PropTypes.string.isRequired,
+  inactive: PropTypes.bool.isRequired,
+  onNewAllocation: PropTypes.func.isRequired,
+  onEdit: PropTypes.func.isRequired,
+  onDeactivate: PropTypes.func.isRequired,
+  onReactivate: PropTypes.func.isRequired,
+  screenSize: PropTypes.number.isRequired,
+}
+
+const StyledCard = styled(Card)`
+  display: flex;
+  margin-bottom: 2rem;
+  margin-right: ${props => props.screenSize < CARD_STRETCH_BREAKPOINT ? '0.6rem' : '2rem'};
+  box-shadow: 0px 2px 4px rgba(221, 228, 233, 0.5);
+  border: 0px;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding: 12px;
+  height: 264px;
+  width: ${props => props.screenSize < CARD_STRETCH_BREAKPOINT ? '100%' : BASE_CARD_WIDTH + 'px'};
+  transition: all 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
+  :hover {
+    cursor: pointer;
+    box-shadow: 0 9px 10px 0 rgba(101, 148, 170, 0.1);
+  }
+`
+
+const MenuContainer = styled.div`
+  align-self: flex-end;
+  align-items: center;
+`
+
+const ActionLabel = styled.span`
+  margin-left: 15px;
+`
+
+const CardTitle = styled(Text.Block).attrs({
+  size: 'large',
+  weight: 'bold',
+})`
+  font-size: 26px;
+  font-weight: 400;
+  margin-top: 10px;
+  margin-bottom: 5px;
+  text-align: center;
+  color: ${({ theme }) => theme.content};
+  display: block;
+  /* stylelint-disable-next-line */
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
```

Comment:
We shouldn't use browser specific rules, since parcel transpiling process takes care of autoprefixing, and it seriously harms our cross-browser compatibility options.

Is there any alternative to get rid of these `webkit-` rules?